src/objective-c/!ProtoCompiler-gRPCPlugin.podspec

```diff
@@ -82,10 +82,9 @@ Pod::Spec.new do |s|
   s.authors  = { 'The gRPC contributors' => 'grpc-packages@google.com' }
 
   repo = 'grpc/grpc'
-  release = "objective-c-v#{v}"
   file = "grpc_objective_c_plugin-#{v}-macos-x86_64.zip"
   s.source = {
-    :http => "https://github.com/#{repo}/releases/download/#{release}/#{file}",
+    :http => "https://github.com/#{repo}/releases/download/v#{v}/#{file}",
```

Comment:
When we create the release notes here https://github.com/grpc/grpc/releases we'll have to add this zip file (named `grpc_objective_c_plugin-1.0.0-macos-x86_64.zip`). @nicolasnoble, you'll create the release notes, right?